h2o-dist/index.html

```diff
@@ -562,6 +562,7 @@ <h1><span>H<sub>2</sub>O</span></h1>
             </div>
             <h2>Get started with H<sub>2</sub>O in 3 easy steps</h2>
             <p>1. Download H<sub>2</sub>O. This is a zip file that contains everything you need to get started.</p>
+            <p> By default, this setup is open. If you want to secure your installation, follow <a href="https://docs.h2o.ai/h2o/latest-stable/h2o-docs/security.html" target="_blank"> Security Guidelines.</a></p>
```

Comment:
@hannah-tillman can we put it into some kind of "info" box - let's consult @jefffohl maybe there is already a container ready to do that.

Reply:
There doesn't appear to be one - I can add one. Is it OK for me to push changes to this branch @hannah-tillman ? Or would you prefer a PR targeting this branch?

Reply:
@jefffohl Please go ahead and make changes directly here. Thank you so much!